src/setUnion.js

```diff
@@ -0,0 +1,8 @@
+export default function setUnion(arr1, arr2) {
+  const arr = arr1.concat(arr2).sort()
+  for (var i=0; i<arr.length; i++){
+    if (arr[i] === arr[i+1]){
+      arr.splice(arr[i], 1)
+    }
+  } return arr
+}
```

Comment:
I would recommend a different formatting

```js
export default function setUnion(arr1, arr2) {
  const arr = arr1.concat(arr2).sort()
  for (var i = 0; i < arr.length; i++){
    if (arr[i] === arr[i+1]){
      arr.splice(arr[i], 1)
    }
  } 
  return arr
}
```

Also are you sure you can `splice` the array you're looping over? Is that intentional to your logic here?